Makefile

```diff
@@ -118,3 +121,16 @@ dist: clean docs assets
 
 install: clean
 	python setup.py install
+
+pex:
+	ls dist/ka_lite-*.whl | while read whlfile; do pex $$whlfile -o dist/kalite-`unzip -p $$whlfile kalite/VERSION`.pex -m kalite --python-shebang=/usr/bin/python; done
```

Comment:
Thinking about this again, I think you should change it before we start (ab)using cloud systems - after all, unnecessary computation is a waste of resources, whether we pay for them or not :)